docs/api-reference/next/image.md

```diff
@@ -493,10 +494,6 @@ module.exports = {
 }
 ```
 
-> Note on CLS with `layout="raw"`:
```

Comment:
You can’t cause CLS anymore?

Reply:
Since the image now always has height and width, I don't believe you can cause CLS. I'll experiment with it a bit while it's still in experimental to make sure.